index.html

```diff
@@ -18,6 +18,13 @@
                 <div class="bar2"></div>
                 <div class="bar3"></div>
             </nav>
+            <div class="desktop-menu">
+                <ul>
+                    <li><a href="#">portfolio</a></li>
+                    <li><a href="#">About</a></li>
+                    <li><a href="#">Contact</a></li>
+                </ul>
+            </div>
```

Comment:
- [ ] Kindly make sure you stick to the original design as much as possible. The spacing(padding, margin) should match the original design.

### Your design Design :point_down: 
![Screenshot from 2023-05-12 14-52-07](https://github.com/yayaismaail/toolbar-headline/assets/97100091/cf38d3af-7c2d-4aa5-8e1a-d03565e36063)
### Original design :point_down: 
![image](https://github.com/yayaismaail/toolbar-headline/assets/97100091/79f14405-9fcd-425c-88b5-88310db2360f)

Reply:
Fixed.